### PR TITLE
Task s2 create home page

### PR DESCRIPTION
**Pull request by JamesCullen2**
Osama and I have put something together using bootstrap.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -4,9 +4,81 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Totally Pawesome</title>
+    <link rel="stylesheet" type="text/css" href="src/assets/css/vendor/bootstrap.min.css">
```
Comment (ailindoyle) [inline index.php:10]: How is this link to bootstrap different from the php autoload in line 1?

File: src/assets/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (ailindoyle) [inline src/assets/css/styles.css]: whitespace

File: index.php [hunk 3]
```diff
@@ -4,9 +4,81 @@
...
+            <div class="col-xs-8 col-md-4">
+                <a href="#">
+                    <img class="img-thumbnail" src="src/assets/img/cat1.jpg">
+                    <h4>Category1</h4>
+                </a>
+            </div>
+            <div class="col-xs-8 col-md-4">
+                <a href="#">
+                    <img class="img-thumbnail" src="src/assets/img/cat1.jpg">
+                    <h4>Category1</h4>
```
Comment (ailindoyle) [inline index.php]: category numbers all 1 
lines 48, 54, 60, 66, 72, 78

File: index.php (hunk 1 above)
Comment (JamesCullen2) [reply index.php:10]: Bootstrap is now to be given a relative path; the vendor file is for autoload / namespacing only.

File: src/assets/css/styles.css (hunk 2 above)
Comment (JamesCullen2) [reply src/assets/css/styles.css]: Fixed.

File: index.php (hunk 3 above)
Comment (JamesCullen2) [reply index.php]: Done.

**Review by ailindoyle — COMMENTED** [review]
there is a bug between the purple header and the grey sidebar - on smaller browser window size sideways scroll reveals a white space gap between the header and the sidebar, on larger browser window size this gap is always present

**Review by ailindoyle — APPROVED** [review]

**Pull request merged**